drivers/ioexpander/pcf8574.c

```diff
@@ -556,7 +555,8 @@ static int pcf8574_multiwritepin(FAR struct ioexpander_dev_s *dev,
       pin = pins[i];
       DEBUGASSERT(pin < 8);
 
-      gpioinfo("%d. pin=%u value=%u\n", pin, values[i]);
```

Comment:
I think the main idea here was to print `i, pin, value[i]`, since printing I2C address seems to be redundant to me as address is already printed at line 539

Reply:
I agree with you... I will make the changes.

Thanks